fontique/examples/generic_families.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 the Parley Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Font enumeration testing
```

Comment:
Maybe describe what it does specifically here, and terminate your comment with a period. Other than that LGTM.

Reply:
Done. With a full stop. (Though I prefer not to terminate one-line descriptions this way myself.)